src/main/java/org/jabref/gui/ClipBoardManager.java

```diff
@@ -150,7 +146,9 @@ public void setContent(List<BibEntry> entries) throws IOException {
         final ClipboardContent content = new ClipboardContent();
         BibEntryWriter writer = new BibEntryWriter(new FieldWriter(preferencesService.getFieldPreferences()), Globals.entryTypesManager);
         String serializedEntries = writer.serializeAll(entries, BibDatabaseMode.BIBTEX);
-        content.put(DragAndDropDataFormats.ENTRIES, serializedEntries);
```

Comment:
Why do you remove  setting them for the DragAndDropDataFormat here? I think this was also neceessary for drag and drop with groups

Reply:
Drag drop is handled differently.

I checked all usages of the variable carefully.

Example code:

    if (!dragboard.hasContent(DragAndDropDataFormats.GROUP)) {

Reply:
but that is for drag and drop between groups. Not entries

Reply:
I still assume that dragstrop is handled by dragophandler and not by the copy functionality. Dragging is iMho natively supported by JavaFX and we do not need to emulate it via copy and paste. This is how I understood the existing code. I checked drag and drop, too